docs/reference/release-notes/highlights-7.4.0.asciidoc

```diff
@@ -0,0 +1,106 @@
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== API key management
+
+Elasticsearch is introducing https://www.elastic.co/guide/en/elastic-stack-overview/7.4/security-privileges.html[new cluster privileges to manage API keys], allowing cluster administrators to manage everything, and regular users to manage their own keys. Users can create API keys and use them to provide long-term credentials while interacting with Elasticsearch. 
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== TLS settings for email notifications
+
+Notifications may contain sensitive information that must be protected over the wire. This requires that communication with the mail server is encrypted and authenticated properly.
+In 7.4.0, Elasticsearch supports custom https://www.elastic.co/guide/en/elasticsearch/reference/7.4/notification-settings.html#ssl-notification-smtp-settings[TLS settings for email notifications], allowing secure connections to servers with custom security configuration.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Automatic query cancellation
+
+With the release of version 7.4.0 automatic query cancellation will (automatically) terminate queries sent through the _search endpoint when the initiating connection is closed.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Added support for AdoptOpenJDK
+
+AdoptOpenJDK 13 is now supported and will ship with Elasticsearch as the pre-bundled JDK (as always, if you want to bring your own JDK, you can still do so by setting `JAVA_HOME` before starting Elasticsearch.) The availabiliity of a notarized AdoptOpenJDK package (per the new requirements for software running on macOS Catalina) will faciltate notarization of Elasticsearch for continued support on macOS. 
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Regression analysis - Experimental
+
+https://www.elastic.co/guide/en/elastic-stack-overview/7.4/dfa-regression.html[Regression analysis] is a machine learning process for estimating the relationships among a number of feature variables and a dependent variable, then making further predictions based on the described relationship.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== New vector distance functions for document script scoring - Experimental
+
+We are introducing two new vector similarity measurements - Manhattan distance (L1 norm) and Euclidean distance (L2 norm). Like the dot product and cosine similarity, the Euclidean and Manhattan distances are provided as https://www.elastic.co/guide/en/elasticsearch/reference/7.4/query-dsl-script-score-query.html#vector-functions[predefined Painless functions] so that they may be incorporated with other query elements as part of a https://www.elastic.co/guide/en/elasticsearch/reference/7.3/query-dsl-script-score-query.html[script_score] query.
```

Comment:
```suggestion
Two experimential similarity measurements—
Manhattan distance (L1 norm)
and Euclidean distance (L2 norm)—
have been added.
Like the dot product and cosine similarity,
the Euclidean and Manhattan distances are provided as <<vector-functions,predefined Painless functions>>
so that they may be incorporated with other query elements
as part of a <<query-dsl-script-score-query,script_score>> query.
```
Replaced hyphen with em dash.